test/e2e/network/proxy.go

```diff
@@ -309,9 +310,10 @@ var _ = common.SIGDescribe("Proxy", func() {
 					RestartPolicy: v1.RestartPolicyNever,
 				}}, metav1.CreateOptions{})
 			framework.ExpectNoError(err, "failed to create pod")
-
+			framework.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(f.ClientSet, "agnhost", f.Namespace.Name))
+			framework.Logf("Pod is running now.")
 			err = wait.PollImmediate(podRetryPeriod, podRetryTimeout, checkPodStatus(f, "test=response"))
-			framework.ExpectNoError(err, "Pod didn't start within time out period")
+			framework.ExpectNoError(err, "Pod did not response with correct status")
```

Comment:
we have the same problem in the next test then

https://github.com/kubernetes/kubernetes/pull/107375/files#diff-53d8ea346826e0686770fbee5d0e48c16e21d146b8184cf40ebd9915ab627398R409

Reply:
Update the next test as well.